### DISCUSSION TITLE
Integration of ORT with azure pipelines?

### DISCUSSION THREAD

**Discussion by andife** [oss-review-toolkit/ort#7463] [Q&A]
Hi,
does anyone have an integration of ORT with azure pipelines?
Or recommendations on how to run it there?

**Comment by sschuberth** [comment]
The (unmaintained) Azure integration we had was removed as part of https://github.com/oss-review-toolkit/ort/pull/5050. While I feel it was overly complicated, you may want to use that as a starting point.

**Reply by tsteenbe** [reply]
@andife  I currently developing a new one based on the existing GitHub and GitLab one, would you be open to help me test it?

**Reply by andife** [reply]
yes, I'm interested. (But I'm not an ORT expert)

**Reply by tom-shephard** [reply]
Did you ever manage to get a working pipeline in Azure?

I am currently trying to do this myself and it is proving tricky!

**Reply by tsteenbe** [reply]
@tom-shephard  I had to stop working ORT for Azure integration as I lost access to Azure when I left my employer. Happy to restart work on it if I can work together with someone with good understanding of setting up Azure pipelines. I know ORT very well but I got stuck it numerous pipeline settings.

**Reply by sujitks** [reply]
Hi @tsteenbe I have good understanding of azure pipelines, let me know if I can help.

**Reply by tsteenbe** [reply]
@sujitks Could you join [ORT Slack](http://slack.oss-review-toolkit.org/) so we chat/call on how we can work together on a ORT for Azure integration?
